src/ImageSharp/Processing/Extensions/ProcessingExtensions.cs

```diff
@@ -150,14 +149,14 @@ public static Image Clone(this Image source, Action<IImageProcessingContext> ope
     /// <exception cref="ObjectDisposedException">The source has been disposed.</exception>
     /// <exception cref="ImageProcessingException">The processing operation failed.</exception>
     /// <returns>The new <see cref="Image"/>.</returns>
-    public static Image Clone(this Image source, Configuration configuration, Action<IImageProcessingContext> operation)
+    public static Image? Clone(this Image source, Configuration configuration, Action<IImageProcessingContext> operation)
     {
         Guard.NotNull(configuration, nameof(configuration));
         Guard.NotNull(source, nameof(source));
         Guard.NotNull(operation, nameof(operation));
         source.EnsureNotDisposed();
 
-        var visitor = new ProcessingVisitor(configuration, operation, false);
+        ProcessingVisitor visitor = new(configuration, operation, false);
```

Comment:
I could have also just changed it to visitor.ResultImage!

Can we always be sure that the visitor has a ResultImage after AcceptVisitor?

Reply:
Let me check

Reply:
I changed the property to a method to make the temporal coupling more obvious. The visitor should always have a result image though.